examples/game_of_life/static/index.html

```diff
@@ -2,11 +2,12 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>Yew • Game of Life</title>
+        <title>Yew Sample App</title>
```

Comment:
I don't think that this change is beneficial.

Reply:
Yes, you are right. I copied the index.html template from your website and forgot to change the title.

Reply:
That's cool 😄. Just wanted to check.

Reply:
```suggestion
        <title>Yew • Game of Life</title>
```